video-morph/src/app/components/dropzone/dropzone.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { FFmpeg } from '@ffmpeg/ffmpeg';
+
+const Dropzone = () => {
```

Comment:
What's the purpose of this component?

Reply:
okay this is supposed to be the main page that loads everything but i have not pushed it yet since i am still trying to fix up some stuff on it, that is where the whole drag and drop occurs and the transcoding funcrions too are there